src/sql-parser/src/parser.rs

```diff
@@ -1212,7 +1305,15 @@ impl<'a> Parser<'a> {
         if let Some(token) = self.peek_token() {
             debug!("get_next_precedence() {:?}", token);
 
-            match &token {
+            let tok = match &token {
+                Token::Keyword(OPERATOR) if self.peek_nth_token(1) == Some(Token::LParen) => {
+                    self.peek_operator_expr()
```

Comment:
My read of this is that it is returning the precedence for the token inside the `OPERATOR`, and I'm not convinced this is correct. See https://www.postgresql.org/docs/14/sql-syntax-lexical.html#SQL-PRECEDENCE:

> When a schema-qualified operator name is used in the OPERATOR syntax, the OPERATOR construct is taken to have the default precedence shown in Table 4.2 for “any other operator”. This is true no matter which specific operator appears inside OPERATOR().